src/java.base/share/classes/jdk/internal/foreign/NativeMemorySegmentImpl.java

```diff
@@ -93,6 +97,7 @@ public Object unsafeGetBase() {
     }
 
     @Override
+    @ForceInline
```

Comment:
I'm not sure this is needed -- this is an "accessor" method - C2 typically inline those. But, do we depend on `maxAlignMask` for reinterpret?